ui/tests/acceptance/secrets/backend/aws/aws-configuration-test.js

```diff
@@ -190,30 +196,27 @@ module('Acceptance | aws | configuration', function (hooks) {
       await runCmd(`delete sys/mounts/${path}`);
     });
 
-    test('it should save lease AWS configuration', async function (assert) {
-      assert.expect(3);
+    test('it does not save lease AWS configuration if root configuration errored on save', async function (assert) {
+      assert.expect(1);
       const path = `aws-${this.uid}`;
       await enablePage.enable('aws', path);
 
       this.server.post(configUrl('aws', path), () => {
-        assert.false(
-          true,
-          'post request was made to config/root when no data was changed. test should fail.'
+        assert.ok('post request was made to config/root when no data was changed. test should fail.');
```

Comment:
When we last discussed this in the team patterns discussion we agreed to avoid using `assert.ok` can this be updated to `assert.true`